kubectl-openebs/cli/command/describe/pool_info.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package describe
+
+import (
+	"github.com/openebs/openebsctl/client"
+	"github.com/openebs/openebsctl/kubectl-openebs/cli/util"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	poolInfoCommandHelpText = `
+This command fetches information and status of the various aspects 
+of the cStor Pool Instance and its underlying related resources in the provided namespace.
+If no namespace is provided it uses default namespace for execution.
+$ kubectl openebs describe pool [cspi-name] -n [namespace]
+`
+)
+
+const (
+	cStorPoolInstanceInfoTemplate = `
+{{.Name}} Details :
+----------------
+Name             : {{.Name}}
+Hostname         : {{.HostName}}
+Size             : {{.Size}}
+Free Capacity    : {{.FreeCapacity}}
+Read Only Status : {{.ReadOnlyStatus}}
+Status	         : {{.Status}}
+RAID Type        : {{.RaidType}}
+
+`
+
+	blockDevicesInfoFromCSPI = `
+Block Device Details :
+----------------
+Name     : {{.Name}}
+Capacity : {{.Capacity}}
+State   : {{.State}}
```

Comment:
- Indentation is a bit off here.
You can ignore the above comment if the below comments are addressed.